bitnami/fluentd/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd
-version: 2.2.3
+version: 2.2.4
```

```suggestion
version: 2.3.0
```

This should be a minor version instead of a patch because we are adding new features.